AcadeMeData/models.py

```diff
@@ -199,3 +199,52 @@ def __str__(self):
 
     def get_msg_tag(id):
         return MessageTags.objects.get(id=id)
+
+
+class Course(models.Model):
+    course_id = models.IntegerField(primary_key=True, validators=[MinValueValidator(0)], default=0)
+    name = models.CharField(max_length=100)  # Name of the course
+    degree = models.ForeignKey(Degree, on_delete=models.RESTRICT)
+    elective = models.BooleanField(default=False)  # False for mandatory, True for elective
```

Comment:
calling it mandatory is better IMO.

Reply:
Why?

Reply:
It's just more intuitive, and its meaning is clear.

Reply:
Done.